app/services/encryption/password_verifier.rb

```diff
@@ -34,18 +34,24 @@ def initialize
       @kms_client = KmsClient.new
     end
 
-    def digest(password:, user_uuid:)
+    def digest_pair(password:, user_uuid:)
       salt = SecureRandom.hex(32)
       cost = IdentityConfig.store.scrypt_cost
       encrypted_password = encrypt_password(
         password: password, user_uuid: user_uuid, salt: salt, cost: cost,
       )
-      PasswordDigest.new(
+      single_region_digest = PasswordDigest.new(
         encrypted_password: encrypted_password, password_salt: salt, password_cost: cost,
       ).to_s
+
+      RegionalCiphertextPair.new(
+        single_region_ciphertext: single_region_digest,
+        multi_region_ciphertext: nil,
+      )
     end
 
-    def verify(password:, digest:, user_uuid:)
+    def verify(password:, digest_pair:, user_uuid:)
+      digest = digest_pair.single_region_ciphertext
```

Comment:
This digest_pair is the method argument rather than a call to the method above, right? Since we're dealing with naming, to we want to disambiguate? I liked your idea of create_digest_pair.